.github/workflows/tests-robotframework-interactive.yml

```diff
@@ -58,6 +58,8 @@ jobs:
     - name: Install robotframework from pip
       run: python -W ignore -m pip install "robotframework>=4.0"
       if: contains(matrix.name, '-pip-v4')
+    - name: Install robotremoteserver via pip
```

Comment:
I don't think you're testing the robotremoteserver with the interactive console (which is ok), but then, this shouldn't be needed (i.e.: this file is for the tests related to the interactive console).